tests/nodeos_eos_evm_test.py

```diff
@@ -1120,14 +1120,68 @@ def get_block(num):
             Utils.Print("  Found ERROR in EOS EVM RPC log: ", line)
             foundErr = True
 
+    Utils.Print("Switching to Savanna")
+    cluster.activateInstantFinality()
+    time.sleep(2.0)
```

Comment:
`sleep` is not needed. By default `activateInstantFinality()` has `waitForFinalization=True` which means it will not return until the transition is complete. Since you are not really doing anything except making sure the test works with Savanna (not really during transition), you can just pass `activateIF=True` to `cluster.launch` and it will transition to Savanna as part of the bootstrap process.

Reply:
thanks. will remove the sleep() call. This test will cover the switching case with the eos-evm-node connected to SHIP, so i guess it would be better not to activate Savanna in cluster.launch().